src/main/java/org/dmfs/oauth2/client/tokens/ImplicitGrantAccessToken.java

```diff
@@ -131,4 +134,10 @@ public OAuth2Scope scope() throws ProtocolException
     {
         return new OptionalParameter<>(SCOPE, mRedirectUriParameters).value(mScope);
     }
+
+    @Override
+    public Optional<CharSequence> extraParameter(final String parameterName) throws ProtocolException
```

Comment:
For testing purposed you can create an `ImplicitGrantAccessToken` like so:
```
new ImplicitGrantAccessToken(
    new LazyUri(new Precoded("http://localhost?state=1&key=value")),
    new EmptyScope(),
    "1",
    new Duration(1,1,0));
```
The state parameter has to be present, otherwise the constructor will throw.

Reply:
For this, I created the `ImplicitGrantAccessTokenTest` in the same package as the `JsonAccessTokenTest`. I made sure to put your existing Copyright text above the package statement.